### PR TITLE
docs: pubsub: removed beta notices

### PR DESCRIPTION
**Pull request by callmehiphop**
Relates to https://github.com/GoogleCloudPlatform/gcloud-common/issues/15

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Woo! Could you sort the readme now to have Pub/Sub appear alphabetically. The order is Stable > Beta > Alpha

**Comment by callmehiphop** [comment]
> Woo! Could you sort the readme now to have Pub/Sub appear alphabetically. The order is Stable > Beta > Alpha

Done!

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This client supports the following Google Cloud Platform services:
 * [Google Cloud Datastore](#google-cloud-datastore)
 * [Google Cloud DNS](#google-cloud-dns)
 * [Google Cloud Storage](#google-cloud-storage)
-* [Google Cloud Pub/Sub](#google-cloud-pubsub-beta) (Beta)
+* [Google Cloud Pub/Sub](#google-cloud-pubsub)
```
Comment (stephenplusplus) [inline README.md]: This just needs to go up one! Sorry!

Comment (callmehiphop) [inline README.md]: derp. 

done!

**Comment by stephenplusplus** [comment]
Thanks!

**Pull request merged**